test/e2e/set/scylladbmonitoring/scylladbmonitoring_webhooks.go

```diff
@@ -0,0 +1,76 @@
+package scylladbmonitoring
+
+import (
+	"context"
+	"fmt"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	scyllav1alpha1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1alpha1"
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/storage/names"
+)
+
+var _ = g.Describe("ScyllaDBMonitoring webhook", func() {
+	f := framework.NewFramework("scylladbmonitoring")
+
+	g.It("should forbid invalid requests", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
+		defer cancel()
```

Comment:
nit: use `g.SpecContext` and set timeout on the ginkgo node
nit: I'd also remove the test timeout altogether, it's limited by the suite anyway